src/ui/public/state_management/state_monitor.js

```diff
@@ -50,6 +55,17 @@ function stateMonitor(state, defaultState) {
   };
 
   return {
+    setDefault(defaultState) {
+      // update the originalState and apply ignoredProps
+      if (defaultState) {
```

Comment:
Maybe it makes sense to just remove this condition. Why would you even call this method without passing in a defaultState?